.travis.yml

```diff
@@ -17,6 +17,8 @@ addons:
     packages:
     - jq
 jobs:
+  allow_failures:
```

Comment:
Yeah, I was considering adding this back in today with @Maxime2. Happy to have it back in now, but our goal should be its removal.

Reply:
I agree, unfortunately the failing tests are multiplying. Pull request #97 went out of its way to break more stuff, haha :)
Also wanted to make sure that the testnet would run successfully this time.